lake/api/api.go

```diff
@@ -40,6 +40,17 @@ type Interface interface {
 	UpdateIndex(ctx context.Context, names []string, pool ksuid.KSUID, branchName string) (ksuid.KSUID, error)
 }
 
+func OpenLake(ctx context.Context, u string) (Interface, error) {
+	uri, err := storage.ParseURI(u)
+	if err != nil {
+		return nil, err
+	}
+	if IsLakeService(uri) {
+		return OpenRemoteLakeWithURI(ctx, uri)
+	}
+	return OpenLocalLakeWithURI(ctx, uri)
+}
+
```

Comment:
This isn't being used, nor can it with the cli since it needs to set the auth store on the connection.

Reply:
@mattnibs: I think it's for use in example code.